scripts/schema.sql

```diff
@@ -30,7 +39,23 @@ CREATE TABLE IF NOT EXISTS `sprint_runs` (
     `run_id` INT NOT NULL AUTO_INCREMENT,
     `start_time` TIMESTAMP(3) NULL,
     `end_time` TIMESTAMP(3) NULL,
+    `play_time` FLOAT,
```

Comment:
Could this be a computed property instead? Or does it also take into account the load times

Reply:
This takes into account the load times! 

https://github.com/wikispeedruns/wikipedia-speedruns/pull/348/files#diff-f143ca6d25ffa21c96acaa8486f99b2a0c02bbd3d9e89c5c89195e678ac2b27eR164-R166